roctracer/PKGBUILD

```diff
@@ -12,10 +12,10 @@ options=(!staticlibs strip)
 _git='https://github.com/ROCm-Developer-Tools/roctracer'
 source=("roctracer-rocm-$pkgver.tar.gz::$_git/archive/rocm-$pkgver.tar.gz"
         'deprecated_string_split.patch')
-sha256sums=('ac4a1d059fc34377e906071fd0e56f5434a7e0e4ded9db8faf9217a115239dec'
+sha256sums=('f47859a46173228b597c463eda850b870e810534af5efd5f2a746067ef04edee'
             '466d34242462c0b2016a82cc6bd90780b578e7a62d6f255b5ba8047fb6925bea')
 _dirname="$(basename "$_git")-$(basename "${source[0]}" ".tar.gz")"
-
+-
```

Comment:
typo?

Reply:
Yes, this included in the output of `sha256sum < some_file` and I forgot to delete it.